addon/components/lio-popover.js

```diff
@@ -65,13 +65,12 @@ export default Component.extend(ParentComponentMixin, ChildComponentMixin, Activ
     return position && position.replace('-', ' ');
   }).property('renderedPosition'),
 
-  position: computed({
-    get: function() {
-      return positions[0];
-    },
-    set: function(key, value) {
+  position: computed(function(key, value) {
+    if (arguments.length > 1) {
       assert(Ember.String.fmt("Position must be one of %@", [ JSON.stringify(positions) ]), positions.contains(value));
       return value;
+    } else {
```

Comment:
You shouldn't need this `else`, since the `if` returns early. `if`-`return`-`else`-`return` is sort of anti-pattern